packages/client/hooks/useCallbackRef.ts

```diff
@@ -1,8 +1,8 @@
 import {useCallback, useRef} from 'react'
 import useForceUpdate from './useForceUpdate'
 
-const useCallbackRef = () => {
-  const ref = useRef<HTMLDivElement | null>(null)
+const useCallbackRef = <T = HTMLDivElement>() => {
```

Comment:
@nickoferrall just a little typescript sugar so we can use this for more than just div elements